upi/gcp/template/01_vpc.py

```diff
@@ -0,0 +1,66 @@
+def GenerateConfig(context):
```

Comment:
I think `upi/gcp/01_vpc.py` is probably fine (dropping `template/`).  The AWS examples use `upi/aws/cloudformation/...`, but I don't think `cloudformation/` adds anything to those paths; it's just an extra tab I have to hit while completing them.  Do you expect many non-template resources in the GCP directory such that we need to shard the templates out into an additional subdirectory?

Reply:
Done.